feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added locally in the feature1 branch
```

Comment:
it's really good that you have decided to write this line of code.

Reply:
Thanks for your comment danishgambit!